### ISSUE TITLE
Psycopg3 | 'Connection' object has no attribute 'server_version'

### ISSUE THREAD

**Issue by LorenzoProd** [M1ha-Shvn/django-pg-bulk-update#97]
In Psycopg v3, the attribute 'server_version' has moved inside the `ConnectionInfo` class.

I've solved the exception in the title by changing the following line in the file src/django_pg_bulk_update/compatibility.py at line 166 from:
`num = conn.cursor().connection.server_version`
to:
`num = conn.cursor().connection.info.server_version`

Has anyone else encountered the same issue?

**Comment by M1ha-Shvn** [comment]
Thanks for the report. I'll have a look in a while, or you can speed it up with a pr

**Comment by LorenzoProd** [comment]
I don't have access rights to the repo.
How about the following?

```py
    try:
        # server_version moved to ConnectionInfo class in psycopg 3.0
        num = conn.cursor().connection.info.server_version
    except AttributeError:
        # server_version is directly available in the Connection class in psycopg 2.x
        num = conn.cursor().connection.server_version
  ```

**Comment by M1ha-Shvn** [comment]
You don't need access. It's a public repo. You can clone it and then make a pr from your clone repo branch to this repo master branch

**Comment by M1ha-Shvn** [comment]
https://github.com/M1ha-Shvn/django-pg-bulk-update/pull/98#issue-2380715677

**Comment by M1ha-Shvn** [comment]
Released [v3.7.3](https://github.com/M1ha-Shvn/django-pg-bulk-update/releases/tag/v3.7.3)
